fun.js

```diff
@@ -0,0 +1,86 @@
+    picture: './images/Claire.png',
+    text: 'Claire Andrewartha is a passionate and dedicated lead teacher of Technology and Humanities with almost 20 years teaching experience in the primary and secondary classroom.',
+  },
+  {
+    id: 'Rahat',
+    name: 'Rahat Arain',
+    post: 'Rahat co-founded an Independent school in the Northern suburbs of <br>Melbourne',
+    picture: './images/Rahat.png',
+    text: 'Rahat co-founded an Independent school in the Northern suburbs of Melbourne. Having spent the last 15 years in school leadership, she has dedicated her efforts to exploring how students can become the best versions of themselves.',
+  },
+  {
+    id: 'Shyam',
+    name: 'Shyam Barr',
+    post: 'Dr Shyam Barr is an education consultant, coach and speaker',
+    picture: './images/Shyam.png',
+    text: 'Dr Shyam Barr is an education consultant, coach and speaker, helping educators foster students’ metacognition and self-regulated learning (SRL). He is a co-host of the podcast Educate to Self-Regulate and a TEDx Speaker (TEDx Canberra’s 2022 event – Impact)',
+  },
+  {
+    id: 'Steve',
+    name: 'Steve Brophy',
+    post: ' Director of Digital Transformation at Ivanhoe Grammar School',
+    picture: './images/Steve.png',
+    text: 'Steve Brophy is a dedicated and experienced leader in the field of education. With a passion for human-centred learning and a deep understanding of the potential for technology to drive positive change.',
+  },
+  {
+    id: 'Sam',
+    name: 'Sam Cobb',
+    post: 'Works at the Alannah and Madeline Foundation as an eSmart Advisor <br>in the Policy and Prevention team',
+    picture: './images/Sam.png',
+    text: 'Sam has always been people-focused, having spent a decade in stakeholder engagement roles within universities, developing relationships with industry, education partners and schools through Australia and South-East Asia.',
+  },
+];
+
+const personalities = document.getElementById('speakers');
+
+function displayPersonality() {
+  let displayPer = '';
+  for (let i = 0; i < 6; i += 1) {
+    displayPer += `<div class='sara'>
+                      <div class="d-flex flex-column">
+                      <img class="picture-b" src="./images/wb-bg.jpg" alt="backgroung design image">
+                      <img class="picture-s" src="${speakerInfo[i].picture}" alt="backgroung design image">
+                      </div>
+                                      <div class='speaker-info'>
+                                          <h2 class='speaker-name'>${speakerInfo[i].name}</h2>
+                                          <h3 class='speaker-post'>${speakerInfo[i].post}</h3>
+                                          <hr class='speaker-hr'>
+                                          <p class='speaker-data'>${speakerInfo[i].text}</p>
+                        </div>
+                    </div>
+                      
+        `;
+  }
+  personalities.innerHTML = displayPer;
+}
```

Comment:
- [ ] Please make sure that the elements don't overlap in the "Speakers" section on the mobile view.

![image](https://user-images.githubusercontent.com/62290545/225334672-8b2eeee9-3762-423c-ad21-509f8e9fddb9.png)

Reply:
Thanks for your progressive feedback, I will surely consider these changes in the mobile version